scripts/windows/build-agent.ps1

```diff
@@ -11,7 +11,7 @@ $imageTag = "rancher/rancher-agent:$version"
 
 pushd $rootPath
 Write-Host "Running agent build"
-if ($version.endswith("-1803")) {
```

Comment:
I think we need to support building agent for both `1803` and `1809`

Reply:
`1809` host can package `1803` container with Hyper-V mode.